cedar-drt/create_corpus.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+FUZZ_TARGET=abac-type-directed
+TIMEOUT="15m" # 15m = 900s
```

Comment:
small suggestion to default to 1hr instead. we've had real bugs only discovered after multiple hours of fuzzing. most interesting bugs discovered by fuzzing were not discovered in the first 15 min IIRC

Reply:
I think we decided on 15mins because we wanted to make regenerating the corpus easier. People were running scripts to modify the corpus rather than regenerate, which meant the code to generate the corpus could get out of sync with the actual corpus.

IMO it makes sense for to leave this at 15mins for now and to continue to rely on the nightly runs to find deeper bugs. I'll try to get some mutation testing experiments done "soon," and there I'll check what percentage of mutations are caught by a 15min vs 1hr corpus. If the difference is significant, then I'd obviously support increasing this number.